env-dev.yml

```diff
@@ -1,13 +1,13 @@
 name: payu-dev
 channels:
   - accessnri
-  - coecms
   - conda-forge
 dependencies:
+  - python>=3.10
```

Comment:
I think it should be `==` so it only changes when we want it to. I can see arguments both ways, but lets stick with `==` and modify is required.

Reply:
Yeah at least its a source of truth of what python version it is using. I've updated it in [76d244b](https://github.com/ACCESS-NRI/payu-condaenv/pull/33/commits/76d244bd0625c27a33f09751effef04434b23c96)